ci/verify-build.sh

```diff
@@ -67,8 +68,31 @@ test_target() {
     # Test again without default features, i.e. without "std"
     $cmd --no-default-features
     $cmd --no-default-features --features extra_traits
+
+    # For freebsd targets, check with the different versions we support if on
+    # nightly or stable
+    case "$rust-$target" in
+        stable-*freebsd*) do_freebsd_checks=1 ;;
+        nightly-*freebsd*) do_freebsd_checks=1 ;;
+    esac
+    
+    if [ "${do_freebsd_checks:-}" = "1" ]; then
```

Comment:
I would write it like this, but that's just a matter of style. 
```suggestion
    if [ -n "$do_freebsd_checks" ]; then
```

Reply:
Ah it still needs the fallback since it may be unset, but I changed to `-n`

Reply:
The quotes are the key to making it work when the variable is unset.  You don't actually need the `:-`